src/voice.ts

```diff
@@ -205,26 +192,30 @@ export class Voice extends Element {
     }
   }
 
-  // We use softmax to layout the tickables proportional to the exponent of
-  // their duration. The softmax factor is used to determine the 'linearness' of
-  // the layout.
-  //
-  // The softmax of all the tickables in this voice should sum to 1.
+  /**
+   * We use softmax to layout the tickables proportional to the exponent of
+   * their duration. The softmax factor is used to determine the 'linearness' of
+   * the layout.
+   *
+   * The softmax of all the tickables in this voice should sum to 1.
+   */
   setSoftmaxFactor(factor: number): this {
     this.options.softmaxFactor = factor;
     return this;
   }
 
-  // Calculate the sum of the exponents of all the ticks in this voice to use as the denominator
-  // of softmax.
-  reCalculateExpTicksUsed(): number {
+  /**
+   * Calculate the sum of the exponents of all the ticks in this voice to use as the denominator
+   * of softmax.
```

Comment:
Can this all fit on one line? :-)

Overall, this style of commenting looks good!

Over time, I imagine we will add the `@param ` as needed, to explain any params that might not be obvious.

Reply:
@param is already used with `mode`